File: Api/EasyUi/src/models/panel/panel.ts
export interface PanelCreateDto {
    userId?: number;
    name: string;
    url: string;
    userName: string;
    password: string;
    ssl?: boolean;
}

export interface PanelUpdateDto {
    name?: string;
    url?: string;
    userName?: string;
    password?: string;
    ssl?: boolean;
}

export interface PanelDto {
    id: number;
    name: string;
    url: string;
    userName: string;
    userId: number;
    password: string;
    sshCertificateKey: string;
    xrayPort: number;
    apiPort: number;
    serverPort: number;
    token: string;
    panelIpAddress: string;
}

File: Domain/Entities/Panel.cs
using System.ComponentModel.DataAnnotations.Schema;
using Domain.Common;

namespace Domain.Entities;

[Table("Panels", Schema = "Commerce")]

public class Panel : BaseEntity<long>
{
    public string Name { get; set; }
    public string UserName { get; set; }
    public string Password { get; set; }
    
    public DateTime LastUsedAt { get; set; } = DateTime.UtcNow;
    public string Url { get; set; }
    public long UserId { get; set; }
    public int? XrayPort { get; set; }
    public int? ApiPort { get; set; }
    public int? InboundPort { get; set; }
    public string? CertificateKey { get; set; }
    public bool SSL { get; set; }
    
    //Relations by other entities
    public ICollection<Instance> Instances { get; set; }
    public User User { get; set; }
    public string Token { get; set; }
}